packages/epics/src/governance/components/document-badges.tsx

```diff
@@ -0,0 +1,34 @@
+import { Badge, type BadgeProps } from '@hypha-platform/ui';
+import { cn } from '@hypha-platform/lib/utils';
+
+export interface BadgeItem extends BadgeProps {
+  value: string | number;
+}
+
+interface DocumentBadgesProps {
+  badges: BadgeItem[];
+  isLoading?: boolean;
+  className?: string;
+}
+
+export const DocumentBadges = ({
+  badges,
+  isLoading,
+  className,
+}: DocumentBadgesProps) => {
+  return (
+    <div className={cn('flex flex-wrap gap-2', className)}>
+      {badges.map((badge, index) => (
+        <Badge
+          key={`${badge.value} ${index}`}
+          className={badge.className}
+          isLoading={isLoading}
+          variant={badge.variant}
+          colorVariant={badge.colorVariant}
+        >
+          {badge.value}
+        </Badge>
+      ))}
+    </div>
+  );
+};
```

Comment:
Seems to be a [duplicate of BadgeList](https://github.com/hypha-dao/hypha-web/blob/381c486bae54ae84192aa03afbe9851321d01aa9/packages/ui/src/molecules/badges-list.tsx#L1-L28)